api/tests/utils/common_functions.py

```diff
@@ -13,6 +13,9 @@ def get_config_file_json_strings(config_sample_path, decamelize=False):
         config_sample_json[section] = config_sample.get_section_dict(section)
 
     if decamelize:
+        """
+        It is strange how humps.decamelize works, beacuse "Source_1" turns into "source__1". 
+        """
```

Comment:
Yes, however, it is strange, because documentation says that will only split on capital letters.
https://github.com/domchristie/humps#humpsdecamelizestring-options